README.adoc

```diff
@@ -1,10 +1,10 @@
-= Address Book (Level 4)
+= ContactSails
```

Comment:
:+1: You are among the few teams who has updated the title of the project :smile: